Java/dp/Rod_Cutting.java

```diff
@@ -0,0 +1,105 @@
+/*Problem Description
+
+You are given a rod of length N inches and an array of prices that contains prices of all pieces of size smaller than N. 
+Determine the maximum value obtainable by cutting up the rod and selling the pieces.
+
+*/
+
+// importing required package 
+import java.util.Scanner;
+
+class Helper {
+	
+    // this method - getValue() will return the final answer to main method 
+    public int getValue(int price[], int n) {
+
+        // m variable will help in getting the size array length by using price length 
+        int m = price.length ;
+        int size[] = new int[m];
+        for(int i = 0 ; i < m  ; i++)
+            size[i] = i + 1;
+
+        // call solve() method for solving the problem
+        int ans = solve(price , size, m , n);
+        return ans ;
+    }
+    
+    //solve() method will act as solving the problem
+
```

Comment:
Remove the extra one line space

Reply:
I have applied the change in commit, see the file.